client/app/src/main/java/com/example/mrides/Notification/InboxAdapter.java

```diff
@@ -6,10 +6,18 @@
 import android.view.ViewGroup;
 import android.widget.TextView;
 
+import com.example.mrides.Activity.InboxActivity;
 import com.example.mrides.R;
 
+import java.util.Map;
+
 public class InboxAdapter extends RecyclerView.Adapter<InboxAdapter.ViewHolder>{
 
+    private Map<String,String> responseBody;
+
+    public InboxActivity(Map<String,String> responseBody){
```

Comment:
There should be a space between the closing parenthesis and opening curly bracket.